scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -223,8 +232,12 @@
       (let [metric-suffix (if (api/synthetic-pod? pod-name) "-synthetic" "")]
         (merge {:waiting-metric-timer
                 (timers/start (metrics/timer (str "k8s-response-time-until-waiting" metric-suffix) name))
+                :waiting-metric-timer-prom-stop-fn
+                (prom/start-timer prom/pod-waiting-duration {:compute-cluster name :synthetic (str (api/synthetic-pod? pod-name))})
                 :running-metric-timer
-                (timers/start (metrics/timer (str "k8s-response-time-until-running" metric-suffix) name))}
+                (timers/start (metrics/timer (str "k8s-response-time-until-running" metric-suffix) name))
+                :running-metric-timer-prom-stop-fn
+                (prom/start-timer prom/pod-running-duration {:compute-cluster name :synthetic (str (api/synthetic-pod? pod-name))})}
```

Comment:
We will really want to separate out the synthetic non-synthetic metrics from each other. I assume that that's easy in dashboarding?

Reply:
Yes, it is easy to select for synthetic being true/false.